### PR TITLE
[IS-50] Queue listener 

### PR DESCRIPTION
**Pull request by Gobonoid**

### CODE REVIEW

File: listener.go [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+	"github.com/JSainsburyPLC/ingestion-sdk/bus"
+	"github.com/JSainsburyPLC/ingestion-validation-service/configuration"
+)
+
+func listen(c *configuration.Configuration) bus.Delivery {
+	session, err := bus.NewSession(c.RabbitMQ.Dial)
+	if err != nil {
+		log.Fatal(err)
+	}
+
```
Comment (patito) [inline listener.go]: log.Fatal() or we could return an error?

File: listener.go [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+		Key:      bus.ValidatorQueueKey,
+		Exchange: bus.ValidatorQueueExchange,
+		NoWait:   false,
+		Args:     nil,
+	}
+	err = session.DeclareBind(bindOptions)
+	if err != nil {
+		log.Fatal(err)
+	}
+
```
Comment (patito) [inline listener.go]: Could we return an error?

File: listener.go [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+		Exclusive: false,
+		NoLocal:   false,
+		NoWait:    false,
+		Args:      nil,
+	}
+
+	delivery, err := session.Consume(consumeOptions)
+	if err != nil {
+		log.Fatal(err)
+	}
```
Comment (patito) [inline listener.go]: could we return an error?

File: listener.go (hunk 1 above)
Comment (Gobonoid) [inline listener.go]: @patito Actually you were right Paulo, it should reDial.

**Comment by Gobonoid** [comment]
@patito Please check if you like my new listener. :)

**Pull request merged**